src/it/resources/org/checkstyle/suppressionxpathfilter/annotationonsameline/SuppressionXpathRegressionAnnotationOnSameLineOne.java

```diff
@@ -0,0 +1,12 @@
+package org.checkstyle.suppressionxpathfilter.annotationonsameline;
+
+public class SuppressionXpathRegressionAnnotationOnSameLineOne
+{
```

Comment:
please move curly brace to previous line to make formatting correct

Reply:
Done